<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>976</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="normal_button">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>330</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Normal Search</string>
    </property>
   </widget>
   <widget class="QPushButton" name="binary_button">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>370</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Binary Search</string>
    </property>
   </widget>
   <widget class="QLabel" name="searchMethod_Label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>260</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Choose your search method</string>
    </property>
   </widget>
   <widget class="QLabel" name="datasetSize_Label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>450</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Input Dataset Size</string>
    </property>
   </widget>
   <widget class="QPushButton" name="generate_button">
    <property name="geometry">
     <rect>
      <x>49</x>
      <y>520</y>
      <width>151</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Generate Dataset</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="size_Text">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>490</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="status_Label">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>160</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Unsorted Array</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>200</y>
      <width>241</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="mergeSort_button">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>470</y>
      <width>171</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Sort Using Merge Sort</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stl_button">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>510</y>
      <width>171</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Sort Using STL Sort</string>
    </property>
   </widget>
   <widget class="QLabel" name="sizeDisplay_Label">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>190</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>The Dataset Size is: </string>
    </property>
   </widget>
   <widget class="QLabel" name="sortDisplay_Label">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>230</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Sorting Using </string>
    </property>
   </widget>
   <widget class="QLabel" name="searchDisplay_Label">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>260</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Using: </string>
    </property>
   </widget>
   <widget class="QTextEdit" name="search_TextEdit">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>320</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="findIt_button">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>370</y>
      <width>91</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Find it!</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
